version/string.go

```diff
@@ -4,37 +4,54 @@
 package version
 
 import (
+	"encoding/json"
 	"fmt"
 	"runtime"
+	"strconv"
 	"strings"
 )
 
-var (
-	// String is displayed when CLI arg --version is used
-	String string
```

Comment:
Do we still want to keep this `String`? We could just export the struct as well rather than implementing a function to grab it